packages/input-checkbox/src/styledComponents/Checkbox.js

```diff
@@ -23,8 +23,8 @@ const checkboxColors = props => {
 const checkboxStyles = props =>
   !props.switch &&
   css`
-    width: ${checkboxSize};
-    height: ${checkboxSize};
+    max-width: ${checkboxSize};
+    max-height: ${checkboxSize};
```

Comment:
I believe the correct prop here would be to set the min-width and min-height, right?

See how it still shrinks with the max props:
<img width="374" alt="Screen Shot 2021-04-09 at 17 25 39" src="https://user-images.githubusercontent.com/17936244/114236978-c539bf80-9958-11eb-972b-6c37c340930f.png">

Reply:
So, I replaced the max with min and at first it worked, but when I opened the webapp on the phone the checkbox was huge. To fix it I added the props `min-width` and `min-height` together with`max-width` and`max-height`